assets/js/components/user-input/UserInputQuestionnaire.js

```diff
@@ -125,6 +126,15 @@ export default function UserInputQuestionnaire() {
 		setActiveSlug( steps[ steps.length - 1 ] );
 	}, [ activeSlugIndex ] );
 
+	useEffect( () => {
+		if ( firstRender ) {
+			setFirstRender( false );
+			return;
+		}
+
+		global.document.getElementById( `googlesitekit-user-input-question-${ activeSlugIndex + 1 }` )?.scrollIntoView( { behavior: 'smooth' } );
+	}, [ activeSlugIndex ] );
```

Comment:
`firstRender|shouldScrollToActiveQuestion` should probably be in the dependency list here as well.

Reply:
Adding `shouldScrollToActiveQuestion` will mean that when we call `setShouldScrollToActiveQuestion( true )` in the effect this will run again and then scroll to the first question on page load.

The existing code works as intended and doesn't warn about dependency issues… I think it's fine 🤷🏻‍♂️